packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -422,6 +422,7 @@
   "Automatic_Translation": "Automatic Translation",
   "Author_Site": "Author site",
   "AutoTranslate": "Auto-Translate",
+  "Auto_Selection": "Auto Selection",
```

Comment:
please remove the old translations from the English file (`Least_Amount` and `Guest_Pool`)